src/schemas/json/cloud-sdk-pipeline-config-schema.json

```diff
@@ -714,54 +685,75 @@
-                            "description": "Maximum heap size of the JVM which runs Fortify SCA.",
-                            "type": "string",
-                            "default": "3000M"
+                        "projectName": {
+                            "description": "The project used for reporting results in SSC",
+                            "type": "string"
                         },
-                        "exclude": {
-                            "description": "File and directories that needs to excluded from the scanning.",
+                        "defaultVersioningModel" : {
+                            "description": "The default project versioning model used in case `projectVersion` parameter is empty for creating the version based on the build descriptor version to report results in SSC, can be one of `'major'`, `'major-minor'`, `'semantic'`, `'full'`",
                             "type": "string",
-                            "default": "**/resources/**/*,**/target/**/*,**/unit-tests/**/*,**/integration-tests/**/*"
+                            "default": "major"
                         },
-                        "skipNgComponents": {
-                            "description": "Skip the node modules from the scanning.",
+                        "buildTool": {
+                            "description": "Scan type used for the step which can be `'maven'`, `'pip'`",
                             "type": "string",
+                            "default": "maven"
+                        },
+                        "autoCreate": {
+                            "description": "Whether Fortify project and project version shall be implicitly auto created in case they cannot be found in the backend",
+                            "type": "boolean",
+                            "default": false
+                        },
+                        "uploadResults": {
+                            "description": "Whether results shall be uploaded or not",
+                            "type": "boolean",
                             "default": true
                         },
-                        "additionalScanOptions": {
-                            "description": "Additional Fortify options such as thread count, alias usage etc.",
-                            "type": "string"
+                        "memory": {
+                            "description": "The amount of memory granted to the translate/scan executions",
+                            "type": "string",
+                            "default": "-Xmx4G -Xms512M"
+                        },
+                        "src": {
+                            "description": "A list of source directories to scan. Wildcards can be used, e.g., `'src/main/java/**/*'`. The default value for `buildTool: 'maven'` is ['**/*.xml', '**/*.html', '**/*.jsp', '**/*.js', '**/src/main/resources/**/*', '**/src/main/java/**/*'], for `buildTool: 'pip'` it is ['./**/*'].",
```

Comment:
this could be made more readable by removing some quotes I guess. Note that people will read this without markdown rendering.